core/main/src/Core/Particle.ts

```diff
@@ -159,6 +160,7 @@ export class Particle implements IParticle {
         this.fill = this.shapeData?.fill ?? this.fill;
         this.close = this.shapeData?.close ?? this.close;
         this.particlesOptions = particlesOptions;
+        this.repulse = this.particlesOptions.repulse;
```

Comment:
This copied object is not needed

Reply:
Ummm, without this the repulse data is not loaded into the particle?

Reply:
`this.particlesOptions.repulse` is already there, there's no need to have an alias

Reply:
Ah, nvm.